canonical-websites/maas.io
# Update site to Vanilla 1.8.0 and fix nav styles

## Done

- Updated Vanilla from 1.6.3 to 1.8.0
- Fixed styling for the nav bar
- Included some styles from https://github.com/canonical-websites/maas.io/pull/274 that will be used in subsequent PRs

## QA

- Pull code, run `./run`
- Check that the navigation looks good on small and large screens
- Note that the page content itself will look broken. This is expected, and will be fixed in subsequent PRs.

## Issue / Card

Fixes #277
